tox.ini

```diff
@@ -8,27 +8,21 @@ max-line-length = 99
 [testenv]
 deps =
     -rrequirements-test.txt
-    django18: Django~=1.8.0
-    django19: Django~=1.9.0
-    django110: Django~=1.10.0
     django111: Django~=1.11.0
 commands = python manage.py test tests/
 
-[testenv:flake8]
+[testenv:py37-flake8]
 skip_install = True
 deps = flake8
 commands = flake8 major_event_log tests setup.py
 
 [travis]
 python =
-    2.7: py27, flake8
-    3.4: py34
+    2.7: py27
     3.5: py35
     3.6: py36
+    3.7: py37, flake8
 
 [travis:env]
 DJANGO =
-    1.8: django18, flake8
-    1.9: django19
-    1.10: django110
-    1.11: django111
+    1.11: django111, flake8
```

Comment:
Does flake8 need to be defined here? It shouldn't need any version of Django installed for a flake8 check.

Reply:
well..You are right. But, I have put flake8 here 'cos we had something similar in the master branch code ` 1.8: django18, flake8`. so didn't remove it.

Reply:
Yeah, tox-travis is what is making this setup a bit more confusing than what we've seen with repos that don't use it. I'm not sure what benefit we're even getting at this point. And one downside is that we lose some separation of envs in Travis. Like if you view the past results, you'll see flake8 doesn't have it's own build, it's grouped together with the py37 tests. @ldko, @madhulika95b, do you think we should just ditch tox-travis?

Reply:
Yea I think we can use tox instead of tox-travis here and get rid of weird combo envs like `1.8: django18, flake8`

Reply:
I think that's fine.

Reply:
Looks like I have to use tox-travis in travis.yml file, 'cos using tox is redundantly running all my tests in each version of Python, causing py36, py37 tests to fail because those interpreters can't be found.

Reply:
well, it won't work with the _current_ setup of envs, we'll have to change things up. we could use previous apps as a reference